### ISSUE TITLE
package-lock.json file...

### ISSUE THREAD

**Issue by JPOak** [olefredrik/FoundationPress#1266]
So I have't used an updated version of FoundationPress since this merge  #1260  . It merged a bunch of stuff to the package-lock.json file. I did an npm update to a version before this and the files are still totally different. Was the merge in  #1260 intentional? Needed?

**Comment by colin-marshall** [comment]
It was not necessary to change `package-lock.json` in that PR since nothing in `package.json` was modified.

**Comment by JPOak** [comment]
@colin-marshall I guess there is no harm in it....or should it be reverted do you think? It looks like it may just add things unique to a custom project. Maybe I am thinking too much into it.

**Comment by colin-marshall** [comment]
I think we're ok as long as it doesn't cause any issues. I think it will get updated whenever a dependency is updated, regardless of if we change `package.json`. 

https://medium.com/@Quigley_Ja/everything-you-wanted-to-know-about-package-lock-json-b81911aa8ab8

**Comment by JPOak** [comment]
@colin-marshall I actually deleted the lock file and regenerated it by running npm install. It was still different than the past version and the current version. I guess I am over thinking it and it all doesn't matter since the package.json file take precedence with versions.

I suppose the only  drawback might be if there is things that don't exist in the package.json file, but do exist in the lock file I believe they will be downloaded, which will make the node_modules file larger.

I will close this.

**Comment by Aetles** [comment]
So, I know it's an old and closed issue and this is not my project and I'm sort of ranting… but it would be a bit more re-assuring if PRs were checked more carefully and commits only were made with the relevant files instead of "well, this file shouldn't have been committed but ¯\_(ツ)_/¯ hopefully it won't cause any trouble". It's not the first time this happens.

**Comment by colin-marshall** [comment]
@Aetles I agree. Looking back, neither @JPOak or myself had anything to do with the PR in question that got merged. The lockfile had not caused any problems after the merge so I was more or less saying if a problem pops up from it we will deal with it, but I didn’t foresee that happening from the research I did on the lock file. The lockfile was a newer thing at the time and we didn’t really have any rules in place on when to merge it and when to ignore it.

Did a problem arise with this? Just curious why you chose this specific issue to point this out.

PS Not my project either, but your input is always welcome as far as I’m concerned. You have helped it a ton along the way and I appreciate how you advocate that we uphold/raise our standards.

**Comment by Aetles** [comment]
No, no problem or anything, all my projects so far is based on older versions of FoundationPress anyway. But from time to time I read through commits to see what's changed and why (and if there's something I should backport to my projects). 

This time, if I may digress, I was actually checking out why jquery-migrate was added to `enqueue-scripts.php` and from there I saw the child theme change (https://github.com/olefredrik/FoundationPress/pull/1260) and got curious about that (there were some debate way back if FP should support child themes or not, since it was meant purely as a starter theme). So that's why I ended up here, lurking in old issues… :)
